src/components/AlefSymbol.tsx

```diff
@@ -16,30 +16,32 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { CSSProperties } from 'react'
 import styled from 'styled-components'
 
 import alefSymbol from '@/images/alef.svg'
 
 interface AlefSymbolProps {
+  color?: string
   className?: string
-  style?: CSSProperties
 }
 
-const AlefSymbol = ({ className, style }: AlefSymbolProps) => (
+const AlefSymbol = ({ className, color }: AlefSymbolProps) => (
   <span className={className}>
     <HiddenForCopying>&nbsp;ALPH</HiddenForCopying>
-    <AlefSymbolSVG style={{ backgroundColor: 'currentColor', ...style }} />
+    <AlefSymbolSVG color={color} />
   </span>
 )
 
-export default AlefSymbol
+export default styled(AlefSymbol)`
+  height: 1em;
+`
 
 const HiddenForCopying = styled.span`
   font-size: 0;
 `
 
-const AlefSymbolSVG = styled.span`
+const AlefSymbolSVG = styled.span<{ color?: string }>`
```

Comment:
The `color` prop is not used anymore, so it's not needed

Reply:
From what I understand the `currentColor` property basically refers to the value of the `color` property. Since there is no `color` property defined here, how does this work?

Reply:
It uses color inheritance if I'm not mistaken :)